README.md

```diff
@@ -29,25 +29,25 @@ We built AvalancheJS with ease of use in mind. With this library, any Javascript
 * Add Validators and Delegators to the Primary Subnetwork by staking AVAX
 * Create a Subnetwork
 * Administer a local node
-* Retrieve Avalanche network information from a node
+* Retrieve Camino network information from a node
 
 ### Requirements
 
-AvalancheJS requires Node.js LTS version 14.16.0 or higher to compile.
+CaminoJS requires Node.js LTS version 14.16.0 or higher to compile.
 
 ### Installation
 
-Avalanche is available for install via `yarn`:
+Camino is available for install via `yarn`:
 
-`yarn add avalanche`
+`yarn add @c4tplatform/camino`
 
 You can also pull the repo down directly and build it from scratch:
 
 `yarn build`
 
-This will generate a pure Javascript library and place it in a folder named "web" in the project root. The "avalanche.js" file can then be dropped into any project as a pure javascript implementation of Avalanche.
+This will generate a pure Javascript library and place it in a folder named "web" in the project root. The "avalanche.js" file can then be dropped into any project as a pure javascript implementation of Camino.
```

Comment:
"camino.js" file
 also, the current version when I run build the web folder is not generated we need to fix that